Formula/ntfy.rb

```diff
@@ -0,0 +1,68 @@
+require "language/node"
+
+class Ntfy < Formula
+  desc "Send push notifications to your phone or desktop via PUT/POST"
+  homepage "https://ntfy.sh/"
+  url "https://github.com/binwiederhier/ntfy/archive/v1.29.1.tar.gz"
+  sha256 "da6c63312ed2f44c1684899c153065d75fbfd45801bf9aa7a701aa1d852f9a2b"
+  license any_of: ["Apache-2.0", "GPL-2.0-only"]
+  head "https://github.com/binwiederhier/ntfy.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "go" => :build
+  depends_on "node" => :build
+  uses_from_macos "sqlite" => :build
+
+  def install
+    cd "web" do
+      system "npm", "install", *Language::Node.local_npm_install_args
+      system "npm", "run", "build", *Language::Node.local_npm_install_args
+      mv "build/index.html", "build/app.html"
+    end
+    rm_rf "server/site"
+    mv "web/build", "server/site"
+    rm "server/site/config.js"
+    rm "server/site/asset-manifest.json"
+    mkdir_p "server/site/static"
+    mkdir_p "server/docs"
+    touch "server/docs/index.html"
+    touch "server/site/app.html"
```

Comment:
Is there no upstream system to manage this?

Reply:
it does have `make`, but when running `make`, it did not even detect npm. also the `make` upstream has the npm in the target so I guess that does not work.

I can try to work with the upstream project devs to make changes if needed?

Reply:
so just to clarify, npm install, run/build has to happen outside make and the file/folder wrangling inside make?

Reply:
I think that would be preferable, yes.